tests/unit_tests/config/test_common.py

```diff
@@ -13,12 +13,27 @@
 #  limitations under the License.
 # -------------------------------------------------------------------------------------------------
 
+import unittest
+
 import msgspec.json
+import pandas as pd
 
 from nautilus_trader.config import ImportableConfig
+from nautilus_trader.config.common import CUSTOM_DECODINGS
+from nautilus_trader.config.common import CUSTOM_ENCODINGS
+from nautilus_trader.config.common import msgspec_decoding_hook
+from nautilus_trader.config.common import msgspec_encoding_hook
+from nautilus_trader.config.common import register_json_decoding
+from nautilus_trader.config.common import register_json_encoding
+from nautilus_trader.core.uuid import UUID4
+from nautilus_trader.model.data import BarType
+from nautilus_trader.model.identifiers import ComponentId
+from nautilus_trader.model.identifiers import InstrumentId
+from nautilus_trader.model.objects import Price
+from nautilus_trader.model.objects import Quantity
 
 
-class TestConfigCommon:
+class TestConfigCommon(unittest.TestCase):
```

Comment:
Thanks for all the tests!

We're moving away from test classes in favor of free functions with `pytest` fixtures as needed. Again, I can refactor this post merge.